packages/evm/src/precompiles/index.ts

```diff
@@ -55,12 +55,12 @@ interface PrecompileAvailabilityCheckTypeEIP {
   type: PrecompileAvailabilityCheck.EIP
   param: number
 }
-
-const ripemdPrecompileAddress = '0000000000000000000000000000000000000003'
+const BYTES_19 = '00000000000000000000000000000000000000'
```

Comment:
I'm not 100% sure I love this as an optimization.  Feels a tiny bit like a bridge too far for tree-shaking at the expense of readability but can live with it if it gains that much for us.